### DISCUSSION TITLE
Made a repo for auto-building with Sqlar enabled

### DISCUSSION THREAD

**Discussion by Qriist** [utelle/SQLite3MultipleCiphers#183] [Show and tell]
Hey again! Hope you're doing well. :)

I'm just sharing a small repo I made for (eventually) auto-building MC with SQLar enabled. It's not quite fully automated yet, nor is it anything special, but it does do the most important task of getting everything built and zipped up in one click. lol.

https://github.com/Qriist/SqlarMultipleCiphers

**Comment by utelle** [comment]
Thanks for the notification.

Actually, the _SQLite3MC_ shell binaries provided in [SQLite3MC releases](https://github.com/utelle/SQLite3MultipleCiphers/releases) have _SQLar_ support enabled. However, the DLLs don't have it enabled by default, because there was no demand for that feature, and besides that support can be enabled easily for the DLLs as well in the provided build files.

Nevertheless, your repo may find interested developers.

**Reply by Qriist** [reply]
> Actually, the _SQLite3MC_ shell binaries provided in [SQLite3MC releases](https://github.com/utelle/SQLite3MultipleCiphers/releases) have _SQLar_ support enabled. 

I noticed that, it's neat!


> However, the DLLs don't have it enabled by default, because there was no demand for that feature

No worries at all! I am under no illusions that I'm operating in anything but the nichest of niche spaces. 😅 


> Nevertheless, your repo may find interested developers.

Always the dream, eh? :)
